dank-client/client.go

```diff
@@ -37,6 +42,28 @@ func (d *Client) resolve() string {
 	return srvclient.MaybeSRV(d.hostname)
 }
 
+var quoteEscaper = strings.NewReplacer("\\", "\\\\", `"`, "\\\"")
```

Comment:
Not sure honestly. I just copied and pasted the code from https://golang.org/src/mime/multipart/writer.go#L116